src/libs/ReportActionsUtils.ts

```diff
@@ -588,7 +588,7 @@ function isReportActionDeprecated(reportAction: OnyxEntry<ReportAction>, key: st
         CONST.REPORT.ACTIONS.TYPE.REIMBURSEMENT_SETUP_REQUESTED,
         CONST.REPORT.ACTIONS.TYPE.DONATION,
     ];
-    if (deprecatedOldDotReportActions.includes(reportAction.actionName as ReportActionName)) {
```

Comment:
Should we use a different type here or is it fine to no longer assert the type?

Reply:
The lint error that occurs if you add this back is:

```
ESLint: This assertion is unnecessary since it does not change the type of the expression.(@typescript-eslint/no-unnecessary-type-assertion)
```

Which is pretty self-explanatory. The type of `reportAction.actionName` is already `ReportActionName`, so the type assertion has no effect.

Same for the few other places where I fixed lint errors of this type by removing `as XYZType`

Reply:
Ah got it, that makes sense, thank you